awx/main/tasks/jobs.py

```diff
@@ -1655,8 +1591,6 @@ def pre_run_hook(self, inventory_update, private_data_dir):
                 sync_task = project_update_task(job_private_data_dir=private_data_dir)
                 sync_task.run(local_project_sync.id)
                 local_project_sync.refresh_from_db()
-                inventory_update.inventory_source.scm_last_revision = local_project_sync.scm_revision
-                inventory_update.inventory_source.save(update_fields=['scm_last_revision'])
```

Comment:
This looks like it may need something added back in. Jobs have an `scm_revision` field to track which revision it ran. We may want to add back in a similar field for inventory updates.

Reply:
would it be better to change this to `scm_revision` to be consistent with `Job.scm_revision`?

Reply:
I'm hitting some similar questions in https://github.com/ansible/awx/pull/12356. Jobs and inventory updates logically should have a multitude of things in common (like this) but it was never fully thought out. My short answer is "yes", it should be consistent, but the broader UI impact and things probably need to get some thought.